test/hotspot/jtreg/compiler/floatingpoint/TestSubNodeFloatDoubleNegation.java

```diff
@@ -55,7 +55,9 @@ public static void assertResults() {
     @Test
     // Match a generic SubNode, because scalar Float16 operations are often
     // performed as float operations.
-    @IR(counts = { IRNode.SUB, "2" })
+    @IR(counts = { IRNode.SUB, "2" }, applyIfPlatform = {"riscv64", "false"})
+    @IR(counts = { IRNode.SUB, "2" }, applyIfCPUFeature = {"zfh", "false"})
+    @IR(counts = { IRNode.SUB, ">= 2" }, applyIfCPUFeature = {"zfh", "true"})
```

Comment:
Just a small nit: I find the following expresses the intention of the test more precisely

```suggestion
    @IR(counts = { IRNode.SUB_HF, "2" }, applyIfCPUFeature = {"zfh", "true"})
```

Reply:
Make sense, fixed!